about.html

```diff
@@ -0,0 +1,212 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.4.0/css/all.min.css"
+        crossorigin="anonymous" referrerpolicy="no-referrer" />
+    <link href="https://fonts.googleapis.com/css2?family=Lato:wght@100;300;400;700;900&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="about.css">
+    <title>About</title>
+</head>
+
+<body>
+    <!--Header Section-->
+    <section id="headerSection" class="header-section">
+        <div class="top-bar">
+            <div class="top-bar-container">
+                <ul class="social-media-list">
+                    <li class="social-media-item">
+                        <i class="social-media-icon fa-brands fa-facebook-f"></i>
+                    </li>
+                    <li class="social-media-item">
+                        <i class="social-media-icon fa-brands fa-twitter"></i>
+                    </li>
+                </ul>
+                <ul class="login-bar">
+                    <li class="login-item">English</li>
+                    <li class="login-item">Mypage</li>
+                    <li class="login-item">Log Out</li>
+                </ul>
+            </div>
+        </div>
+        <header class="header-container">
+            <a href="index.html" class="logo-link">
+                <img src="assets/festival.svg" alt="Logo" class="logo-image">
+            </a>
+            <nav class="navbar">
+                <i id="menuOpenIcon" class="fa-solid fa-bars"></i>
+                <ul class="navbar-list">
+                    <li class="navbar-item">
+                        <a href="index.html" class="navbar-link active">Home</a>
+                    </li>
+                    <li class="navbar-item">
+                        <a href="index.html#program" class="navbar-link">Program</a>
+                    </li>
+                    <li class="navbar-item">
+                        <a href="index.html#speakersSection" class="navbar-link">Chefs</a>
+                    </li>
+                    <li class="navbar-item">
+                        <a href="#partners" class="navbar-link">Partners</a>
+                    </li>
+                    <li class="navbar-item">
+                        <a href="#footer" class="navbar-link">Contact</a>
+                    </li>
+                    <li class="navbar-item ticket-item">
+                        <a href="" class="navbar-link ticket-link">Buy tickets</a>
+                    </li>
+                </ul>
+            </nav>
+        </header>
+    </section>
```

Comment:
- [ ] Same here as pointed out earlier :point_up_2: It would be better to wrap this in a `header` tag.